examples/pulse_control/2_Native_gate_calibrations.ipynb

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
<p>1) Now that you have access to a...</p><p> </p><p>2) We use <s>here</s> a&nbsp;<code>draw</code>&nbsp;method&nbsp;</p><p> </p><p>3) applied on <s>a</s> qubit 0. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>